junit5/core/src/main/java/org/jboss/arquillian/junit5/JunitEventTestRunnerAdaptor.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.arquillian.junit5;
+
+import org.jboss.arquillian.core.api.annotation.ApplicationScoped;
+import org.jboss.arquillian.core.spi.ManagerBuilder;
+import org.jboss.arquillian.test.impl.EventTestRunnerAdaptor;
+import org.jboss.arquillian.core.impl.loadable.LoadableExtensionLoader;
+
+/**
+ * Extend the default EventTestRunnerAdaptor to allow for setting method parameters.
+ */
+public class JunitEventTestRunnerAdaptor extends EventTestRunnerAdaptor {
+
+    public static JunitEventTestRunnerAdaptor build() {
+        ManagerBuilder builder = ManagerBuilder.from()
+            .extension(LoadableExtensionLoader.class);
+        return new JunitEventTestRunnerAdaptor(builder);
+    }
+
+    JunitEventTestRunnerAdaptor(ManagerBuilder builder) {
+        super(builder);
+    }
+
+    void setMethodParameters(MethodParameters methodParameters) {
+        getManager().bind(ApplicationScoped.class, MethodParameters.class, methodParameters);
```

Comment:
Maybe the binding scope should be TestScoped

Reply:
That would be my guess as well. I'll pull this down and do some more testing though.